dvc/command/repro.py

```diff
@@ -90,13 +90,6 @@ def add_parser(subparsers, parent_parser):
         help="Reproduce only single data item without recursive dependencies "
         "check.",
     )
-    repro_parser.add_argument(
-        "-c",
```

Comment:
This breaks backward compatibility :slightly_frowning_face: If we want to do things right, I would make `-c` an alias for `-C` for now.